File: kargahEvenOrOdd.java
import java.util.Scanner;

public class kargahEvenOrOdd {
    public static void main(String[] args) {
        int a;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter an integer: ");
        a = sc.nextInt();
        if (a%2 == 0)
            System.out.println("Even");
        else
            System.out.println("Odd!!!!");
        sc.close();
    }
}

File: Kargah_even_or_odd.cpp
#include <iostream>
using namespace std;

bool even(int n);

int main()
{

    int n;
    cin >> n;
    if (even(n))
        cout << "even";
    else
        cout << "odd";

    return 0;
}

bool even(int n)
{
    return (n % 2 == 0);
}